packages/commerce-sdk-react/src/hooks/Test/query.ts

```diff
@@ -0,0 +1,158 @@
+
+    return useQuery(
+        apiOptions,
+        {
+            queryKey,
+            ...queryOptions
+        },
+        {
+            client,
+            method,
+            requiredParameters
+        }
+    )
+}
+/**
+ * A hook for `ShopperBaskets#getShippingMethodsForShipment`.
+ * Gets the applicable shipping methods for a certain shipment of a basket.
+ * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-baskets?meta=getShippingMethodsForShipment} for more information about the API endpoint.
+ * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperbaskets.shopperbaskets-1.html#getshippingmethodsforshipment} for more information on the parameters and returned data type.
+ * @returns An object describing the state of the request.
+ */
+export const useShippingMethodsForShipment = (
+    apiOptions: Argument<Client['getShippingMethodsForShipment']>,
+    queryOptions: UseQueryOptions<DataType<Client['getShippingMethodsForShipment']>>
+): UseQueryResult<DataType<Client['getShippingMethodsForShipment']>> => {
+    const requiredParameters = ['organizationId', 'basketId', 'shipmentId', 'siteId'] as const
+    const queryKey = ['TODO']
+    const {shopperBaskets: client} = useCommerceApi()
+    const method = client.getShippingMethodsForShipment.bind(client)
+
+    return useQuery(
+        apiOptions,
+        {
+            queryKey,
+            ...queryOptions
+        },
+        {
+            client,
+            method,
+            requiredParameters
+        }
+    )
+}
+/**
+ * A hook for `ShopperBaskets#getTaxesFromBasket`.
+ * This method gives you the external taxation data set by the PUT taxes API. This endpoint can be called only if external taxation mode was used for basket creation. See POST /baskets for more information.
+ * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-baskets?meta=getTaxesFromBasket} for more information about the API endpoint.
+ * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperbaskets.shopperbaskets-1.html#gettaxesfrombasket} for more information on the parameters and returned data type.
+ * @returns An object describing the state of the request.
+ */
+export const useTaxesFromBasket = (
+    apiOptions: Argument<Client['getTaxesFromBasket']>,
+    queryOptions: UseQueryOptions<DataType<Client['getTaxesFromBasket']>>
+): UseQueryResult<DataType<Client['getTaxesFromBasket']>> => {
+    const requiredParameters = ['organizationId', 'basketId', 'siteId'] as const
+    const queryKey = ['TODO']
+    const {shopperBaskets: client} = useCommerceApi()
+    const method = client.getTaxesFromBasket.bind(client)
+
+    return useQuery(
+        apiOptions,
+        {
+            queryKey,
+            ...queryOptions
+        },
+        {
+            client,
+            method,
+            requiredParameters
+        }
+    )
+}
```

Comment:
Question: this PR has a title `simplify hooks` and no description. Are we modifying only existing hooks or are we adding new ones?


When the PR title says "simplify" and the diff shows `+658` that's confusing to me

<img width="176" alt="image" src="https://user-images.githubusercontent.com/47546998/215903364-4280345d-cb53-4364-9140-ccf4a66f1db2.png">

Reply:
1. Because it's a WIP that I threw up at 5PM before vanishing for a day. Figured it's better to have code with no context than no code at all.
2. It's a WIP. Note that the folder the new hook is in is `Test`; the actual hooks haven't been touched yet. I'm looking more for feedback on design than doing the actual code changes.